parquet-hadoop/src/main/java/org/apache/parquet/hadoop/InternalParquetRecordWriter.java

```diff
@@ -154,6 +157,7 @@ private void checkBlockSizeReached() throws IOException {
 
   private void flushRowGroupToStore()
       throws IOException {
+    recordConsumer.flush();
```

Comment:
page flushing is handled by the ColumnWriter, basically when you ask ColumnWriter to write sth, it will cut the pages automatically